data-model/fdo-profiles/0.1.0/README.md

```diff
@@ -0,0 +1,14 @@
+## FDO Profile schemas
+
+This folder contains a collection of FAIR Digital Object Profiles (FDO Profiles) specific to DiSSCo data model elements. 
+Each folder represents a distinct profile for various digital objects like annotations, digital specimens, media objects, organisations. We also have profiles for DOI and Handle kernel, for mapping, machine annation service 
+(mas), and digital object tomstone pages. These profiles are crucial for implementing FAIR principles and ensuring machine-actionability.
+
```

Comment:
Maybe mention that the profiles rely on each other? Objects given a handle (e.g. mas, mapping, annotations) depend on the handle profile, and objects given a doi (digital specimens, media objects) rely on the handle profile and the doi profile.